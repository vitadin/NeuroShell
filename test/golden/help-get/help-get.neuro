# Test help information for get command
\help[get]