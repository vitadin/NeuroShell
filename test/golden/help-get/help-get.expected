Command: get
Description: Get a variable
Usage: \get[var] or \get var
Parse Mode: Key-Value (supports [key=value] syntax)
Examples:
  \get[var] or \get var
  \get[option=value]